test_functions.s

```diff
@@ -0,0 +1,21 @@
+.global ret42
```

Comment:
Stop including assembly files. I added a commit that has these in C so you can get rid of this